TSC-Charter.md

```diff
@@ -17,60 +17,47 @@ In the case of the Node.js project, it is delegated to the Node.js
 
 TSC memberships are not time-limited. There is no maximum size of the TSC.
-The size is expected to vary in order to ensure adequate coverage of important
-areas of expertise, balanced with the ability to make decisions efficiently.
-The TSC must have at least four members.
+The TSC must have at least four voting members.
 
 There is no specific set of requirements or qualifications for TSC
-membership beyond these rules. The TSC may add additional members to the
+membership beyond these rules. The TSC may add additional voting members to the
 TSC by a standard TSC motion. A TSC member may be removed from the
-TSC by voluntary resignation, by a standard TSC motion, or in accordance to the
-participation rules described below.
+TSC by voluntary resignation or by a standard TSC motion.
 
-Changes to TSC membership should be posted in the agenda, and may be suggested
-as any other agenda item.
-
-No more than one-fourth of the TSC members may be affiliated with the
-same employer. If removal or resignation of a TSC member, or a change of
-employment by a TSC member, creates a situation where more than
-one-fourth of the TSC membership shares an employer, then the situation
+No more than one-fourth of the TSC voting members may be affiliated with the
+same employer. If removal or resignation of a TSC voting member, or a change of
+employment by a TSC voting member, creates a situation where more than
+one-fourth of the TSC voting membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
-TSC members affiliated with the over-represented employer(s).
-
-The TSC may, at its discretion, invite any number of non-voting observers to
-participate in the public portion of TSC discussions and meetings.
+TSC voting members affiliated with the over-represented employer(s).
 
 The TSC shall meet regularly using tools that enable participation by the
 community (e.g. weekly on a Google Hangout On Air, or through any other
 appropriate means selected by the TSC). The meeting shall be directed by
 the TSC Chairperson. Responsibility for directing individual meetings may be
-delegated by the TSC Chairperson to any other TSC member. Minutes or an
+delegated by the TSC Chairperson to any other TSC voting member. Minutes or an
 appropriate recording shall be taken and made available to the community
 through accessible public postings.
 
-TSC members are expected to regularly participate in TSC activities.
+TSC voting members are expected to regularly participate in TSC activities.
 
-A TSC member is automatically removed from the TSC if, during a 3-month period,
-all of the following are true:
+A TSC voting member is automatically converted to a TSC regular member if,
+during a 3-month period, all of the following are true:
 
 * They attend fewer than 25% of the regularly scheduled meetings.
-* They do not participate in any TSC votes.
+* They do not participate in three consecutive TSC votes.
```

Comment:
This is sort of more nit-picky but I'm having trouble following whether this 3-month removal only applies if there are 3 votes within a 3 month period or if it's "attend fewer than 25% of regularly scheduled meetings in a 3-month period AND do not participate in three consecutive TSC votes"?

Reply:
I think we should remove the 3-month part entirely as well as the meeting requirement (as previously discussed and as Geoffrey points out above), and just have three consecutive votes be it. I suppose we could add some text to protect against three votes happening in one day or something like that, but I'm personally not too worried about that. (Votes are usually open for a week or two anyway, or at least a few days. And they are infrequently concurrent.)

Reply:
I would change it so votes are taken asynchronously (not during meetings) and the pools are open for at least a week.

Reply:
> This is sort of more nit-picky but I'm having trouble following whether this 3-month removal only applies if there are 3 votes within a 3 month period or if it's "attend fewer than 25% of regularly scheduled meetings in a 3-month period AND do not participate in three consecutive TSC votes (in any given time-frame)"?

I think this is resolved with the change pushed just now, but if I'm wrong, please leave another comment.